src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -20,7 +20,7 @@
 
 logger = azlogging.get_az_logger(__name__)
 
-regex = re.compile('/subscriptions/(?P<subscription>[^/]*)/resourceGroups/(?P<resource_group>[^/]*)'
+regex = re.compile('/subscriptions/(?P<subscription>[^/]*)/resource[gG]roups/(?P<resource_group>[^/]*)'
```

Comment:
I'd rather not. Making it case-insensitive here is kind of a hack b/c that's what arm explorer is using. This is a case-by-case workaround, not a case desensitizing in general.